Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:12.5 as base
+FROM debian:12.5 AS base
```

Comment:
It seems like part of changes from https://github.com/WikiTeq/Taqasta/pull/226/files was also included into this PR, suggesting rebasing PRs frequently to prevent this